_vector-search/tutorials/neural-search-tutorial.md

```diff
@@ -0,0 +1,877 @@
+      {
+        "_index": "my-nlp-index",
+        "_id": "2",
+        "_score": 0.6839765,
+        "_source": {
+          "text": "A wild animal races across an uncut field with a minimal amount of trees.",
+          "id": "1775029934.jpg"
+        }
+      },
+      {
+        "_index": "my-nlp-index",
+        "_id": "3",
+        "_score": 0.3007,
+        "_source": {
+          "text": "People line the stands which advertise Freemont 's orthopedics , a cowboy rides a light brown bucking bronco .",
+          "id": "2664027527.jpg"
+        }
+      },
+      {
+        "_index": "my-nlp-index",
+        "_id": "1",
+        "_score": 0.29919013,
+        "_source": {
+          "text": "A West Virginia university women 's basketball team , officials , and a small gathering of fans are in a West Virginia arena .",
+          "id": "4319130149.jpg"
+        }
+      }
+    ]
+  }
+}
+```
+</details>
+
+Instead of specifying the search pipeline in every request, you can set it as a default search pipeline for the index as follows:
+
+```json
+PUT /my-nlp-index/_settings 
+{
+  "index.search.default_pipeline" : "nlp-search-pipeline"
+}
+```
+{% include copy-curl.html %}
+
+You can now experiment with different weights, normalization techniques, and combination techniques. For more information, see the [`normalization-processor`]({{site.url}}{{site.baseurl}}/search-plugins/search-pipelines/normalization-processor/) and [`hybrid` query]({{site.url}}{{site.baseurl}}/query-dsl/compound/hybrid/) documentation.
+
+#### Advanced
+
+You can parameterize the search by using search templates. Search templates hide implementation details, reducing the number of nested levels and thus the query complexity. For more information, see [search templates]({{site.url}}{{site.baseurl}}/search-plugins/search-template/).
+
+### Clean up
+
+After you're done, delete the components you've created in this tutorial from the cluster:
+
+```json
+DELETE /my-nlp-index
+```
+{% include copy-curl.html %}
+
+```json
+DELETE /_search/pipeline/nlp-search-pipeline
+```
+{% include copy-curl.html %}
+
+```json
+DELETE /_ingest/pipeline/nlp-ingest-pipeline
+```
+{% include copy-curl.html %}
+
+```json
+POST /_plugins/_ml/models/aVeif4oB5Vm0Tdw8zYO2/_undeploy
+```
+{% include copy-curl.html %}
+
+```json
+DELETE /_plugins/_ml/models/aVeif4oB5Vm0Tdw8zYO2
+```
+{% include copy-curl.html %}
+
+```json
+DELETE /_plugins/_ml/model_groups/Z1eQf4oB5Vm0Tdw8EIP2
+```
+{% include copy-curl.html %}
+
+## Further reading
+
+- Read about the basics of OpenSearch semantic search in [Building a semantic search engine in OpenSearch](https://opensearch.org/blog/semantic-search-solutions/).
+- Read about the benefits of combining keyword and semantic search, the normalization and combination technique options, and benchmarking tests in [The ABCs of semantic search in OpenSearch: Architectures, benchmarks, and combination strategies](https://opensearch.org/blog/semantic-science-benchmarks/).
```

Comment:
Delete "the benefits of" so that it isn't read as applying to the second and third list items.